test/jmap/email/set/set_email_method_test.dart

```diff
@@ -261,5 +261,121 @@ void main() {
 
       expect(setEmailResponse!.created![Id('aa1234')], equals(expectedCreated));
     });
+
+    test('set email method and response parsing with header Mdn', () async {
+      final baseOption  = BaseOptions(method: 'POST');
+      final dio = Dio(baseOption)
+        ..options.baseUrl = 'http://domain.com/jmap';
+      final dioAdapter = DioAdapter(dio: dio);
+      dioAdapter.onPost(
+          '',
+          (server) => server.reply(200, {
+              "sessionState": "2c9f1b12-b35a-43e6-9af2-0106fb53a943",
+              "methodResponses": [
+                  [
+                      "Email/set",
+                      {
+                          "accountId": "587a9c5a4a9c0a4d36243b7417700d5383cbbfa25f0909ab7f6f4baaa5bfde9b",
+                          "oldState": "24152b20-4ab0-11ed-88ee-ffc86e0cde67",
+                          "newState": "24152b20-4ab0-11ed-88ee-ffc86e0cde67",
+                          "created": {
+                              "e01": {
+                                  "id": "5e2b02d0-4ab0-11ed-88ee-ffc86e0cde67",
+                                  "blobId": "5e2b02d0-4ab0-11ed-88ee-ffc86e0cde67",
+                                  "threadId": "5e2b02d0-4ab0-11ed-88ee-ffc86e0cde67",
+                                  "size": 600
+                              }
+                          }
+                      },
+                      "0"
+                  ]
+              ]
+          }),
+          data: {
+            "using": [
+              "urn:ietf:params:jmap:core",
+              "urn:ietf:params:jmap:mail",
+              "urn:apache:james:params:jmap:mail:shares"
```

Comment:
`SetEmailMethod` no need for this capability:  `urn:apache:james:params:jmap:mail:shares`